src/metabase/email.clj

```diff
@@ -16,9 +16,16 @@
 ;;; CONFIG
 
 (defsetting email-from-address
-  (deferred-tru "Email address you want to use as the sender of Metabase.")
+  (deferred-tru "The email address you want to use for the sender of emails.")
   :default "notifications@metabase.com")
 
+(defsetting email-from-name
+  (deferred-tru "The name you want to use for the sender of emails."))
+
+(defsetting email-reply-to
+  (deferred-tru "The email address you want the replies to go to, if different from the from address. You can have multiple reply-to email addresses, just separate them with a comma.")
```

Comment:
should we drop the "You can have multiple reply-to email addresses, just separate them with a comma." language until we want to go down that alley? Or is this fine to leave in because everything will work with it, we just don't have a UI to enter it yet?